README.md

```
@@ -12,6 +12,27 @@ From there, users can submit questions and people can vote on how much they agre
 
 This is a React app with a Node.js backend. It uses WebSockets for real-time communication.
 
+## Local development
+
+Start Postgres with Docker Compose:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)